simulators/rpc-compat/src/main.rs

```diff
@@ -144,3 +162,36 @@ dyn_async! {
         }
     }
 }
+
+dyn_async! {
+    async fn test_history_local_content_content_present<'a>(test: &'a mut Test, client: Option<Client>) {
+        let client = client.expect("Client should be available for portal_historySendOffer test");
```

Comment:
That text seems wrong / copy pasted as this is not about `portal_historySendOffer`?

Reply:
The other tests used it, I will fix it in this PR though

Reply:
Oh when #43 gets merged these expects will be gone so I will wait till then, since that will resolve this issue

Reply:
resolved